### ISSUE TITLE
Training data of released V2 model

### ISSUE THREAD

**Issue by Davidyao99** [lpiccinelli-eth/UniDepth#100]
Great Work!

I was wondering for the released UnidepthV2, is this the model trained on only the 16M training images stated in the UnidepthV2 paper? In other words, they are not finetuned on NYU / KITTI?

**Comment by lpiccinelli-eth** [comment]
Thank you for your appreciation!
No, they are not finetuned, but the results for NYU and KITTI tables correspond to the fine-tuned versions, as per typical practice. I can release the weights if you would like to use the fine-tuned version, let me know.

**Comment by Davidyao99** [comment]
I see, thanks for the prompt response! I just wanted to check for evaluation purposes since I ran UnidepthV2 on some sequences in KITTI training sequences, and was wondering if these are within training data of UnidepthV2.

**Comment by jskvrna** [comment]
I would really like to see the fine-tuned weights for the KITTI dataset.

Thank you!

**Comment by yaojin17** [comment]
Hi @lpiccinelli-eth, could you share fine-tuned weights on the KITTI dataset? Thanks in advance!

**Comment by johncwlee** [comment]
Hi @lpiccinelli-eth , any progress on the fine-tuned weights on the KITTI dataset? Thanks!

**Comment by Cats-eat-peppercorns** [comment]
@lpiccinelli-eth @Davidyao99  @johncwlee @yaojin17 @jskvrna hi ！ Could you please tell me if you have achieved the indicators described in the paper? It's very difficult for me to reach this metric using the NYU dataset. My d1 only achieves 93%. I would be very grateful if you could provide me with the detailed hyperparameters.

**Comment by 2020namemyself2020** [comment]
Hi @lpiccinelli-eth, could you share fine-tuned weights on the KITTI dataset? Thanks very much!
